src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt

```diff
@@ -24,20 +24,21 @@ object ParserSetup {
     private const val PYTHON3_PROPERTY = "ac.p3.path"
     private const val PYTHONPARSER_PROPERTY = "gt.pp.path"
 
-    private const val PARSER_REPOSITORY_ZIP_URL =
+    private const val PARSER_REPO_ZIP_URL =
         "https://github.com/JetBrains-Research/pythonparser/archive/master.zip"
     private const val PARSER_ZIP_NAME = "master.zip"
 
     // Relative path in the parser repository
-    private const val REPOSITORY_ROOT_FOLDER = "pythonparser-master"
-    private const val PARSER_RELATIVE_PATH = "$REPOSITORY_ROOT_FOLDER/src/main/python/pythonparser/pythonparser_3.py"
+    private const val PARSER_REPO_FOLDER = "pythonparser-master"
+    private const val PARSER_RELATIVE_PATH = "$PARSER_REPO_FOLDER/src/main/python/pythonparser/pythonparser_3.py"
     private const val INVERSE_PARSER_RELATIVE_PATH =
-        "$REPOSITORY_ROOT_FOLDER/src/main/python/inverse_parser/inverse_parser_3.py"
+        "$PARSER_REPO_FOLDER/src/main/python/inverse_parser/inverse_parser_3.py"
 
-    private const val PARSER_NAME = "pythonparser"
-
-    private val targetParserPath = "${getTmpPath()}/$PARSER_NAME"
-    private val targetInverseParserPath = "${getParserRepositoryPath()}/$INVERSE_PARSER_RELATIVE_PATH"
+    const val PARSER_NAME = "pythonparser"
+    val targetParserPath = "${getTmpPath()}/$PARSER_NAME"
+    val targetInverseParserPath = "${getParserRepoParentFolder()}/$INVERSE_PARSER_RELATIVE_PATH"
+    val parserRepoPath = "${getParserRepoParentFolder()}/$PARSER_REPO_FOLDER"
+    val parserZipPath = "${getParserRepoParentFolder()}/$PARSER_ZIP_NAME"
```

Comment:
Also, you use `targetParserPath`, `parserRepoPath`, and `parserZipPath` only in `ParserSetup` and `PythonSetupTest`. I am not sure about `public` modifier for tests only. Maybe should we create a member function in `ParserSetup` to get a tuple of `parserRepoFile`, `parserZipFile`, and `targetParserFile` for tests? What do you think?